test/UnitTest/System.Web.OData.Test/PublicApi/System.Web.OData.PublicApi.bsl

```diff
@@ -398,6 +398,7 @@ public class System.Web.OData.ETagMessageHandler : System.Net.Http.DelegatingHan
 	public ETagMessageHandler ()
 
 	[
+	DebuggerStepThroughAttribute(),
```

Comment:
Generate this file in Release mode. Only one change is DebuggerStepThroughAttribute that we shouldn't include